pkg/proxy/fixtures/gencerts.sh

```diff
@@ -1,12 +1,21 @@
 #!/bin/bash
 
+set -euo pipefail
+
 if ! [[ "$0" =~ "./gencerts.sh" ]]; then
   echo "must be run from 'fixtures'"
   exit 255
 fi
 
-if ! which cfssl; then
+if ! command -v cfssl; then
   echo "cfssl is not installed"
+  echo 'use: bash -c "cd ../../../tools/mod; go install github.com/cloudflare/cfssl/cmd/cfssl"'
```

Comment:
This comment applies to some places below as well. 

Note that this script may be executed at any directory, so set the absolute directory is better.
```suggestion
  echo 'use: bash -c "cd ${ETCD_ROOT_DIR}/tools/mod; go install github.com/cloudflare/cfssl/cmd/cfssl"'
```

Reply:
This script is only allowed to run in current dir.

```bash
if ! [[ "$0" =~ "./gencerts.sh" ]]; then
  echo "must be run from 'fixtures'"
  exit 255
fi
```

So I was thinking that the user can copy-parse the echo string to install.
If we use ETCD_ROOT_DIR, the user needs to get the root dir path first.

I can add one line to resolve ETCD_ROOT_DIR first.

Reply:
OK, then let's keep it as it is for now.